tpot/builtins/preprocessing.py

```diff
@@ -0,0 +1,93 @@
+
+import sys
+import os
+from importlib import import_module
+
+from sklearn.base import TransformerMixin, BaseEstimator
+from sklearn.feature_extraction.text import TfidfVectorizer, CountVectorizer, HashingVectorizer
+from sklearn.preprocessing import OneHotEncoder
+from importlib import import_module
+from sklearn.compose import ColumnTransformer
+
+import_loader = {
+    'TfidfVectorizer': TfidfVectorizer,
+    'CountVectorizer': CountVectorizer,
+    'HashingVectorizer': HashingVectorizer,
+    'OneHotEncoder': OneHotEncoder
+}
+
+
+
+def load_scoring_function(scoring_func):
```

Comment:
help for getting relative imports working would be appreciated here... `tpot.drivers.load_scoring_function`

Reply:
Hmm, directly importing via `from ..driver import load_scoring_function` will cause some conflicts with 
```python
from .tpot import TPOTClassifier, TPOTRegressor
from ._version import __version__
```

A workaround is to move this function to `tpot/metrics.py` and then add `from ..metrics import load_scoring_function` to `tpot/builtins/preprocessing.py `